eng/versioning/version_client.txt

```diff
@@ -78,7 +78,7 @@ com.azure:azure-e2e;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-identity;1.2.4;1.3.0-beta.3
 com.azure:azure-identity-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-iot-deviceupdate;1.0.0-beta.1;1.0.0-beta.2
-com.azure:azure-messaging-eventgrid;4.0.0;4.1.0-beta.1
+com.azure:azure-messaging-eventgrid;4.0.0;4.1.0
```

Comment:
Out of band release?

Reply:
Yes, communication service team changed the communication service related part of the EventGrid swagger file. New system events are added and a mapping error is fixed.
The communication service will GA on 03/29. We need to GA the updates before service GA.